misc-tools/import-contest.in

```diff
@@ -55,6 +56,40 @@ def import_file(entity: str, files: List[str]):
             last = f'\'{files[-1]}\''
             print(
                 f'Neither {prefix_joined} nor {last} found, skipping {entity} import.')
+    return False
+
+def import_images(entity: str, property: str, filename_regexes: List[str]):
+    """Import images for the given entity. filename_regexes determines what file to import: only
+    files matching the regexes will be considered and if multiple files match the regexes the file
+    with the first matched regex will be used"""
+
+    if not os.path.isdir(entity):
+        return
+    images_per_entity = {}
+    for entity_id in listdir(entity):
+        entity_dir = f'{entity}/{entity_id}'
+        if not os.path.isdir(entity_dir):
+            continue
+        entity_files = listdir(entity_dir)
```

Comment:
```suggestion
        entity_files = sorted(listdir(entity_dir))
```
Not tested, but I think os.listdir has no ordering and I would prefer to get the same images between 2 instances when using the same import script. This might still fail between 2 different filesystems (POSIX/NTFS) but that is fine by me.